lib/src/image_dart_compiling/main.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:scissors/src/image_dart_compiling/image_dart_compiling.dart';
+
+/// Command-line entry point. Takes list of paths to pictures, producing Dart
+/// source on stdout.
+main(List<String> arguments) async {
+  try {
+    String source = await generateDartSource(await Future.wait(arguments.map(
+        (String fileName) async => new ImageInformation(
+            fileName, await new File(fileName).readAsBytes()))));
+    stdout.write(source);
+  } on ConversionException catch (e) {
```

Comment:
Note: this is still not adding value compared to not catching anything, which would both print the stack (which you're not doing) and set some error code. I'd just drop this.